src/pwm.rs

```diff
@@ -0,0 +1,186 @@
+use crate::println;
+use embedded_hal::Pwm;
+use riot_sys::pwm_t;
+
+/// PWM modes
+///
+/// [`PWMMode::CENTER`] exists in RIOT but is marked as "not supported"
+#[derive(Debug)]
+pub enum PWMMode {
+    LEFT,
+    RIGHT,
+    CENTER,
+}
+
+impl PWMMode {
+    /// Converts the rust enum to a c type
+    pub fn to_c(self) -> riot_sys::pwm_mode_t {
```

Comment:
Not sure, just wanted to provide it but maybe it could get just removed to keep the enums pub api clean. If one really needs this its not hard to build it ig.

Reply:
If we find out that there *is* a good reason to have it around, we can still make it pub later.

But with how volatile riot_sys is API-wise compared to riot_wrappers, every sys type we don't expose publicly may save us trouble down the line.